src/sentry/api/endpoints/project_details.py

```diff
@@ -616,19 +610,6 @@ def put(self, request: Request, project) -> Response:
 
         if "dynamicSampling" in result:
             raw_dynamic_sampling = result["dynamicSampling"]
-            if (
```

Comment:
do we want to remove tests?

```
/tests/sentry/api/endpoints/test_project_details.py
```
?
cc @ahmedetefy

Reply:
do you mean create new tests? because there are no tests checking this logic removed

Reply:
Usually, I would say yes that makes sense but looking at that test module, I don't think that branch was tested
@andriisoldatenko which tests specifically are you referring to ?